git_machete/client.py

```diff
@@ -1616,17 +1615,15 @@ def set_fork_point_override(self, branch: LocalBranchShortName, to_revision: Any
         branch_sha = self.__git.get_commit_sha_by_revision(branch)
```

Comment:
Looks there's still a single occurrence left:

```
$ git grep -n 'sys.stdout.write'
git_machete/client.py:1011:        sys.stdout.write(out.getvalue())
```

Reply:
Also! `sys.stderr.write` are still abundant... are they going to be replaced (on this PR?) as well?

Reply:
I've changed `sys.stderr.write()` as well but the above one I couldn't do without getting an error.